install-source.html

```diff
@@ -33,35 +33,27 @@ <h1 id="TOC_1.">1. Build and install from source</h1>
   <h1 id="TOC_2.">2. Prerequisites</h1>
   <h2 id="TOC_2.1.">2.1. Go</h2>
   <p>
-    Install Go 1.2 or newer from <a href="http://golang.org/doc/install" target="_blank">golang.org</a>.
+    Install Go 1.20 or newer from <a href="http://golang.org/doc/install" target="_blank">golang.org</a>.
   </p>
-  <h2 id="TOC_2.2.">2.2. DVCS Clients</h2>
+  <h2 id="TOC_2.2.">2.2. Git</h2>
   <p>
-    Go will need these DVCS clients installed in order to fetch all of Hockeypuck&#39;s
-    package dependencies:
+    Go will need a Git client installed in order to fetch all of Hockeypuck&#39;s
+    package dependencies.
   </p>
-  <ul>
-    <li>Bazaar</li>
-    <li>Git</li>
-    <li>Mercurial</li>
```

Comment:
This reminds me how old this project really is... back in the 1.0 times, some of the Go package dependencies were sourced from `hg` and `bzr` repos when you'd `go get`.

That was only like... 5 years ago... right? :older_man: